src/components/interaction-designer/Block.vue

```diff
@@ -237,6 +231,10 @@ Vue.component('BTooltip', BTooltip)
 
 const LABEL_CONTAINER_MAX_WIDTH = 650
 
+const EXIT_CONNECTED_BG = '#418BCA'
+const EXIT_DISCONNECTED_BG = '#858585'
+const EXIT_HOVER_BG = '#FFECEC'
```

Comment:
We can remove these if we go with approach suggested above!

Reply:
Done!